sdk/wallet-adapter/adapters/unsafe-burner/src/index.ts

```diff
@@ -29,10 +29,17 @@ export class UnsafeBurnerWalletAdapter implements WalletAdapter {
   #provider: JsonRpcProvider;
   #keypair: Ed25519Keypair;
   #signer: RawSigner;
+  #account: ReadonlyWalletAccount;
 
   constructor(network: Connection = devnetConnection) {
     this.#keypair = new Ed25519Keypair();
     this.#provider = new JsonRpcProvider(network);
+    this.#account = new ReadonlyWalletAccount({
+      address: formatAddress(this.#keypair.getPublicKey().toSuiAddress()),
```

Comment:
The formatting here seems wrong, seems like this should be the raw address.

Reply:
This was suppose to be normalise - will remove it thanks!